Assets/Character/AaronTestAssets/Player.cs

```diff
@@ -0,0 +1,60 @@
+﻿using UnityEngine;
+using System.Collections;
+
+
+
+public class Player : MonoBehaviour {
```

Comment:
Player is not a descriptive enough script name. This scripts deals with movement and animation, so the name of the script should reflect that